diff --git a/node_modules/discord.js/src/structures/MessageEmbed.js b/node_modules/discord.js/src/structures/MessageEmbed.js
index eb67b95..4d7c710 100644
--- a/node_modules/discord.js/src/structures/MessageEmbed.js
+++ b/node_modules/discord.js/src/structures/MessageEmbed.js
@@ -429,7 +429,9 @@ class MessageEmbed {
    * @returns {EmbedField[]}
    */
   static normalizeFields(...fields) {
-    return fields
+    if (fields.flat && typeof fields.flat === 'function')
+    {
+      return fields
       .flat(2)
       .map(field =>
         this.normalizeField(
@@ -438,6 +440,18 @@ class MessageEmbed {
           field && typeof field.inline === 'boolean' ? field.inline : false,
         ),
       );
+    }
+    else
+    {
+      return fields
+      .map(field =>
+        this.normalizeField(
+          field && field.name,
+          field && field.value,
+          field && typeof field.inline === 'boolean' ? field.inline : false,
+        ),
+      );
+    }
   }
 }
 
